src/hooks/useNotificationModal.ts

```diff
@@ -31,17 +34,20 @@ export const useNotificationModal = ({
   const slackUsername = userSettingsData?.userSettings?.slackUsername;
   const emailAddress = userData?.user?.emailAddress;
 
-  const [selectedSubscriptionMethod, setSelectedSubscriptionMethod] = useState(
-    ""
-  );
+  const [
+    selectedSubscriptionMethod,
+    setSelectedSubscriptionMethod,
+  ] = useState(() => Cookies.get("subscription-method"));
   // target represents the input value for a subscription method
   const [target, setTarget] = useState<Target>({});
   const [isFormValid, setIsFormValid] = useState(false);
   const [extraFieldErrorMessages, setExtraFieldErrorMessages] = useState<
     string[]
   >([]);
 
-  const [selectedTriggerIndex, setSelectedTriggerIndex] = useState<number>();
+  const [selectedTriggerIndex, setSelectedTriggerIndex] = useState<number>(() =>
+    parseInt(Cookies.get(`${type}-notification-trigger`), 10)
```

Comment:
I'm seeing `NaN` on initial load of this modal, I think since there's no cookie found and then we try and run `parseInt()` on it.

<img width="698" alt="image" src="https://user-images.githubusercontent.com/9298431/133792599-7764c0ab-d47e-405a-a9e6-15f8cc5d0340.png">

Reply:
Thanks for catching i added a default case